libbeat/scripts/cmd/global_fields/main.go

```diff
@@ -29,6 +29,7 @@ import (
 func main() {
 	esBeatsPath := flag.String("es_beats_path", "..", "Path to elastic/beats")
 	beatPath := flag.String("beat_path", ".", "Path to your Beat")
+	toStdout := flag.Bool("stdout", false, "Write output to stdout")
```

Comment:
Personally I'd prefer to standardize on using `-out` to specify the output file in our CLI tools. And make the default be `-` mean output to `os.Stdout` like most unix tools. Then you get the same behavior across all tools. The downside being that you have to write `-out fields.yml` in the common case, but the common case is almost always done via `make` so it's not actually a problem.